biocypher_metta/adapters/ontologies_adapter.py

```diff
@@ -1,168 +1,144 @@
+    SUBCLASS = rdflib.term.URIRef('http://www.w3.org/2000/01/rdf-schema#subClassOf')
+    DB_XREF = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasDbXref')
 
     LABEL = rdflib.term.URIRef('http://www.w3.org/2000/01/rdf-schema#label')
-    RESTRICTION = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#Restriction')
-    TYPE = rdflib.term.URIRef(
-        'http://www.w3.org/1999/02/22-rdf-syntax-ns#type')
-    ON_PROPERTY = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#onProperty')
-    SOME_VALUES_FROM = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#someValuesFrom')
-    ALL_VALUES_FROM = rdflib.term.URIRef(
-        'http://www.w3.org/2002/07/owl#allValuesFrom')
-    NAMESPACE = rdflib.term.URIRef(
-        'http://www.geneontology.org/formats/oboInOwl#hasOBONamespace')
-    EXACT_SYNONYM = rdflib.term.URIRef(
-        'http://www.geneontology.org/formats/oboInOwl#hasExactSynonym')
-    RELATED_SYNONYM = rdflib.term.URIRef(
-        'http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym')
-    DESCRIPTION = rdflib.term.URIRef(
-        'http://purl.obolibrary.org/obo/IAO_0000115')
+    RESTRICTION = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#Restriction')
+    TYPE = rdflib.term.URIRef('http://www.w3.org/1999/02/22-rdf-syntax-ns#type')
+    ON_PROPERTY = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#onProperty')
+    SOME_VALUES_FROM = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#someValuesFrom')
+    ALL_VALUES_FROM = rdflib.term.URIRef('http://www.w3.org/2002/07/owl#allValuesFrom')
+    NAMESPACE = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasOBONamespace')
+    EXACT_SYNONYM = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasExactSynonym')
+    RELATED_SYNONYM = rdflib.term.URIRef('http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym')
+    DESCRIPTION = rdflib.term.URIRef('http://purl.obolibrary.org/obo/IAO_0000115')
 
     PREDICATES = [SUBCLASS, DB_XREF]
     RESTRICTION_PREDICATES = [HAS_PART, PART_OF]
 
-    def __init__(self, type='node', dry_run=False):
+    def __init__(self, write_properties, add_provenance, ontology, type, label, dry_run=False):
+        super().__init__(write_properties, add_provenance)
         self.type = type
+        self.label = label
         self.dry_run = dry_run
-        if type == 'node':
-            self.label = 'ontology_term'
-        elif type == 'edge':
-            self.label = 'ontology_relationship'
-        else:
-            raise ValueError('Invalid type. Allowed values: node, edge')
-
-    def get_graph(self, ontology):
-
-        onto = get_ontology(OntologyAdapter.ONTOLOGIES[ontology]).load()
+        self.graph = None
+        self.cache = {}
+        self.source = None
+        self.source_url = None
+        self.ontology = ontology  # Store the ontology value
+
+    def get_graph(self):
```

Comment:
This method sets the `self.graph` field and then returns it. But it's unnecessary to return the field after setting it.